### PR TITLE
consul 1.0.3

### PR DESCRIPTION
**Pull request by commitay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by commitay** [comment]
```
==> make
go get -u -v github.com/elazarl/go-bindata-assetfs/... github.com/jteeuwen/go-bindata/... github.com/magiconair/vendorfmt/cmd/vendorfmt github.com/mitchellh/gox golang.org/x/tools/cmd/cover golang.org/x/tools/cmd/stringer github.com/axw/gocov/gocov gopkg.in/matm/v1/gocov-html
github.com/elazarl/go-bindata-assetfs (download)
github.com/jteeuwen/go-bindata (download)
github.com/magiconair/vendorfmt (download)
github.com/mitchellh/gox (download)
```
```
Error: No such file or directory - contrib/zsh-completion/_consul
```

**Comment by ilovezfs** [comment]
@commitay we basically never allow `go get` in Homenrew as it's insecure. There's an audit for that when it's used in the install method, but unfortunately we have yet to find a good way to detect its use when it's in a Makefile or build script.

**Comment by commitay** [comment]
Should I turn them into `go_resource` blocks?

**Comment by ilovezfs** [comment]
:) and I guess the usual request upstream for vendoring if you have a moment

File: Formula/consul.rb [hunk 1]
```diff
@@ -16,20 +18,69 @@ class Consul < Formula
...
+    ]
+
+    build_tools.each do |tool|
+      cd buildpath/"src/#{tool}" do
+        system "go", "install"
+      end
+    end
 
-    cd gopath/"src/github.com/hashicorp/consul" do
+    cd buildpath/"src/github.com/hashicorp/consul" do
```
Comment (ilovezfs) [inline Formula/consul.rb]: The buildpath/ is unneeded

File: Formula/consul.rb [hunk 2]
```diff
@@ -16,20 +18,69 @@ class Consul < Formula
...
       system "make"
       bin.install "bin/consul"
       prefix.install_metafiles
-      zsh_completion.install "contrib/zsh-completion/_consul"
```
Comment (ilovezfs) [inline Formula/consul.rb]: did move somewhere?

Comment (commitay) [reply Formula/consul.rb]: Deleted

https://github.com/hashicorp/consul/commit/5143b703e36d98211f28ef0f85d18580bdf1b089

File: Formula/consul.rb [hunk 3]
```diff
@@ -16,20 +18,69 @@ class Consul < Formula
...
+        system "go", "install"
+      end
+    end
 
-    cd gopath/"src/github.com/hashicorp/consul" do
+    cd "src/github.com/hashicorp/consul" do
       system "make"
       bin.install "bin/consul"
       prefix.install_metafiles
-      zsh_completion.install "contrib/zsh-completion/_consul"
```
Comment (ilovezfs) [inline Formula/consul.rb]: Did it move somewhere else?

File: Formula/consul.rb (hunk 2 above)
Comment (ilovezfs) [reply Formula/consul.rb]: Hmm. OK. I guess we'll see if anyone comes knocking asking for us to run that automatically.

**Comment by ilovezfs** [comment]
:white_check_mark: :bowing_man:

**Pull request closed without merging**